[KubeRay][Doc] KubeRay doc migration (1/n) (#38647)
ray-project/ray: doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

(kuberay-rayjob-quickstart)=

# RayJob

> > Note: This is the alpha version of Ray Job Support in KubeRay. There will be ongoing improvements for Ray Job in the future releases.

Reviewer: ```suggestion
> Note: Ray Job Support in KubeRay is in alpha. Ray Job improvements are planned for future releases.
```

Reviewer: Suggest using the {warning} component like the one on the landing page of Ray Serve (which has  {tip} component).

Author: Updated https://github.com/ray-project/ray/pull/38647/commits/21dcebace5896289dcfbe067c7dae54792a0060e

## Prerequisites

* Ray 1.10 or higher
* KubeRay v0.3.0+. (v0.6.0+ is recommended)

## What is a RayJob?

> A RayJob manages 2 things:

Reviewer: ```suggestion
A RayJob manages two aspects:
```
